index.js

```diff
@@ -66,7 +61,18 @@ module.exports = (options = {}) => {
     },
 
     load(plugin) {
+      const robot = createRobot({app, cache, logger, catchErrors: true});
+
+      // Connect the router from the robot to the server
+      server.use(robot.router);
```

Comment:
@anglinb as you suggested in https://github.com/probot/probot/pull/180#issuecomment-319450695, this would ideally prefix the path with the name of the plugin. The challenge is that, depending on how the plugin is loaded, we may or may not have a name readily available.

I'm tempted to punt on the isolation for now and just see how people use this.

WDYT?